src/CoreLib/dependency.fs

```diff
@@ -250,20 +250,20 @@ type JobDependencies() =
     member val EnvVars : List<string*string> = new List<string*string>() with get
     
     /// Job File Dependencies Hash
-    member internal x.JobDependencySHA256() =
+    member internal x.JobDependencyDefaultHash() =
```

Comment:
Yep. I also have a few renamings to make. "Default" is not a very good name. But what else? I think I'll call it "Checksum" instead of Hash, to make it clear what it is, and not be bound to the Fletcher64 algorithm.